src/streams/LogglyStream.js

```diff
@@ -0,0 +1,42 @@
+import { Writable } from 'stream';
+
+import loggly from 'node-loggly-bulk';
+
+class LogglyStream extends Writable {
+    /**
+    * copied from: https://www.loggly.com/blog/node-js-libraries-make-sophisticated-logging-simpler/
+    * Create a new instance of MorganLogglyLogger
+    * @param options {Object}
+    * @param options.token {String} your loggly token
+    * @param options.subdomain {String} your loggly SUBDOMAIN
+    */
+    constructor(options) {
+        // allows use to use any JS object instead of only string or Buffer
+        super({ objectMode: true });
+
+        this.client = loggly.createClient({
+            token: options.token, // your loggly token
+            subdomain: options.subdomain, // your loggly subdomain
+            tags: [options.environment, options.name],
+            json: true,
+        });
+
+        // make sure write function has proper context (this)
+        this._write = this._write.bind(this); // eslint-disable-line
+    }
+
+    _write(message, encoding, callback) { // eslint-disable-line
+        this.client.log(JSON.parse(message));
+        /* XXX callback causing server exit??
```

Comment:
I don't know what this callback does, but calling it seems to cause the server to exit. I did check the data the callback is being called with and we are getting a successful response from loggly...